js/index.js

```diff
@@ -44,10 +44,16 @@ app.controller("IndexController", ["$scope", "$http", 'leafletData', function($s
 
     $scope.currentMarker = marker;
     $("#modal").modal('show');
-    $("#modal").on('click', '#btnClose', function() {
+    $("#modal").on('click', '#btnCancel', function() {
+     marker = new Object();
```

Comment:
Missing space

Reply:
Why do you use `marker = new Object();` here but `marker = {};` below? Aren't you trying to do the same thing?